r2r/core/pipelines/extraction.py

```diff
@@ -0,0 +1,37 @@
+from abc import abstractmethod
+from typing import Iterator, Optional
+
+from ..providers.prompt import PromptProvider
+from ..providers.llm import LLMProvider
+from ..providers.logging import LoggingDatabaseConnection, log_execution_to_db
+from r2r.core import BasicDocument, GenerationConfig
+from r2r.pipelines import Pipeline
```

Comment:
I think the Pipeline class is in .pipeline

Should it not be 

from .pipeline import Pipeline